src/rgw/rgw_rados.h

```diff
@@ -1270,6 +1272,11 @@ struct RGWZoneParams : RGWSystemMetaObj {
     } else {
       reshard_pool = name + ".rgw.reshard";
     }
+    if (struct_v >= 11) {
+      ::decode(groups_pool, bl);
+    } else {
+      roles_pool = name + ".rgw.groups";
```

Comment:
the old style of naming was for older releases that didn't support the namespace feature. as long as these changes don't get backported to kraken or earlier, we can and should use the new namespaced naming style